docs/config.rst

```diff
@@ -405,6 +428,31 @@ Features
 
    .. versionadded:: 2.0
 
+.. attribute:: LP_ENABLE_DISK
+   :type: bool
+   :value: 1
+
+   Display :attr:`LP_MARK_DISK` if the free space on the hard drive hosting the
+   current directory goes below a threshold.
+
+   Thresholds can be stated either:
+   * as a percentage with :attr:`LP_DISK_THRESHOLD_PERC`,
+   * or an absolute number *of kilobytes* with :attr:`LP_DISK_THRESHOLD`.
+   Display will occur if one of the thresholds is met.
```

Comment:
A single newline will not render this as a list. You need a second newline to get list formatting:
```suggestion
   Thresholds can be stated either:

   * as a percentage with :attr:`LP_DISK_THRESHOLD_PERC`,
   * or an absolute number *of kilobytes* with :attr:`LP_DISK_THRESHOLD`.

   Display will occur if one of the thresholds is met.
```